examples/gallery/ui/pages/controls_page.slint

```diff
@@ -38,28 +38,35 @@ export component ControlsPage inherits Page {
     }
 
     GroupBox {
-        title: "CheckBox - SpinBox - ComboBox";
+        title: "CheckBox - SpinBox - ComboBox - Switch";
         vertical-stretch: 0;
 
-        HorizontalBox {
+        HorizontalLayout {
             alignment: start;
+            spacing: 8px;
+
             checkbox := CheckBox {
                 text: checkbox.checked ? "(checked)" : "(unchecked)";
                 checked: true;
                 enabled: !GallerySettings.widgets-disabled;
             }
 
-
-        SpinBox {
-            vertical-stretch: 0;
-            value: 42;
-            enabled: !GallerySettings.widgets-disabled;
-        }
+            SpinBox {
+                vertical-stretch: 0;
+                value: 42;
+                enabled: !GallerySettings.widgets-disabled;
+            }
 
             ComboBox {
                 model: ["Select Something", "From this", "Combobox"];
                 enabled: !GallerySettings.widgets-disabled;
             }
+
+            Switch {
```

Comment:
Perhaps for the widget enable/disable control in the gallery we should use a switch instead :-)